lib/Navigation/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const Navigation = props => (
```

Comment:
This looks almost identical to an existing component `NavTabs`: https://github.com/gathercontent/gather-ui/pull/143/files

I suggest we figure out how to use a single component and add modifiers where required.

Reply:
Merged NavTabs into Navigation